src/components/Profile/Profile.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+
+import s from './Profile.module.css';
+import MyPosts from './MyPosts/MyPosts';
+
+const Profile = () => {
+    return (
+        <div className='content'>
+            <div>
+                <img src='https://a.d-cd.net/870b45as-960.jpg'></img>
```

Comment:
Consider storing this url in a separate const.
Possibly, somewhere in a core folder of your project alongside other image urls.

Reply:
In the future I plan to use the object for avatars, now it's just a workpiece